controllers/admin/AdminStoresController.php

```diff
@@ -415,23 +419,45 @@ protected function postImage($id)
         // Should we generate high DPI images?
         $generate_hight_dpi_images = (bool) Configuration::get('PS_HIGHT_DPI');
```

Comment:
Naming here is confusing. But it probably should be fixed in another PR...


```suggestion
         $generate_high_dpi_images = (bool) Configuration::get('PS_HIGH_DPI');
```

Reply:
@SharakPL I removed this feature from v9 completely, so it's fixed. 😁